tests/test_clockify.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+from clockify_client import Clockify
+
+
+def test_can_be_instantiated() -> None:
+    clockify = Clockify("apikey", "baz.co")
+    assert isinstance(clockify, Clockify)
+
+
+def test_singleton() -> None:
```

Comment:
We should rename this test, you're not testing it's a singleton here, you're honestly testing its not a singleton.

Reply:
Oops.  That's fixed.